src/components/Account.vue

```diff
@@ -5,10 +5,10 @@
                 <img class="account-image" :src="image" @error="showImage = false">
                 <div class="outline"></div>
             </div>
-            <Identicon v-else :address="address"/>
+            <Identicon v-else-if="_isNimiqAddress" :address="address"/>
```

Comment:
What is the `else` case now that this is `else-if`? Just nothing? And when would that happen?

Reply:
This is to avoid trying to display an identicon if the address is not a Nim address, but for example a BTC address.